### PR TITLE
[IMP] doc: add warning about feature only being available in developer mode

### PR DESCRIPTION
**Pull request by Yenthe666**
Description of the issue/feature this PR addresses: Without this warning the enduser has no idea that CDN can only be configured when developer mode is activated.

Desired behavior after PR is merged: The enduser, which has little technical knowledge, knows that he has to activate developer mode before he/she can configure this.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Yenthe666** [comment]
@mart-e for you - should be forward ported to V10 & V11 too.

**Comment by mart-e** [comment]
Hi!

I think we have already discussed that but I don't like notes and warnings 😉 
I think it make the documentation less fluent to read and looks more complex.

Why not just incorporate the info in the above text?
Bonus point if you keep 80 char !

**Comment by Yenthe666** [comment]
@mart-e as requested. Is this ok for you? It is 81 characters if you include the spaces and the dot. If you ignore the dot or spaces I get a bonus point 😉

File: doc/setup/cdn.rst [hunk 1]
```diff
@@ -37,7 +37,7 @@ Step 2: Configure the odoo instance with your zone
 
 In the Odoo back end, go to the :guilabel:`Website Settings`: menu, then
 activate the CDN support and copy/paste your zone URL in the
-:guilabel:`CDN Base URL` field.
+:guilabel:`CDN Base URL` field. This field is only visible and configurable if you have developer mode activated.
```
Comment (mart-e) [inline doc/setup/cdn.rst]: that is not 81, that is exactly 114 characters 😉

Comment (Yenthe666) [reply doc/setup/cdn.rst]: Hey, don't count the text that was already there. 😆 My part is 81 characters, being: " This field is only visible and configurable if you have developer mode activated."
Is it good for you now?

**Comment by mart-e** [comment]
The idea to have a 80 char limit is to have a nice and readble text in my editor.
Too long lines make the readability and the diff smallers.

screenshot from 2018-02-08 15-10-32

Anyway, moved on two lines and merged at 72a8b420d28

**Comment by Yenthe666** [comment]
Thanks a lot.

**Pull request closed without merging**